sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Processor/EventProcessorHost.cs

```diff
@@ -80,6 +80,8 @@ await _checkpointStore.UpdateCheckpointAsync(new EventProcessorCheckpoint()
                 EventHubName = EventHubName,
                 FullyQualifiedNamespace = FullyQualifiedNamespace
             }, checkpointEvent, cancellationToken).ConfigureAwait(false);
+            var leaseInfo = new LeaseInfo(checkpointEvent.Offset, checkpointEvent.SequenceNumber);
```

Comment:
Do we ever request the leaseInfo for partitions not owned by the current processor?

Reply:
If not we should put this value on the https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Processor/EventProcessorHostPartition.cs#L12 as the Checkpoint call goes through it anyway https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Processor/EventProcessorHostPartition.cs#L49

Reply:
Good point - it doesn't look like it.